DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: cellengine
 Type: Package
 Title: CellEngine API Toolkit
-Version: 0.2.0
+Version: 0.3.0
```

Comment:
Let's do the version bump later. The changelog needs a new section added to it and I want to finish the scaleset stuff before calling it "released."

Reply:
:+1: